### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/go-enry/commit/bbe6427b8ac8939526f0a53e50ef0da9a7f69512#commitcomment-80693713

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/go-enry/commit/9f21b24d9d58f3960c7167b479efea13c10b3dae

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: .github/workflows/sync-linguist.yml [hunk 1]
```diff
@@ -84,7 +84,7 @@ jobs:
 
         branch_name="feature/sync-linguist-${{ steps.previous_linguist.outputs.short_commit }}"
         if git rev-parse --quiet --verify $branch_name; then
-          echo "Linuist update branch $branch_name already exists"
+          echo "Linguist update branch $branch_name already exists"
```
Comment (jsoref) [inline .github/workflows/sync-linguist.yml:87]: Brand

**Review by jsoref — COMMENTED** [review]
Most changes suggested by Google Sheets. All fault mine

**Comment by jsoref** [comment]
Note that I've excluded https://github.com/jsoref/go-enry/commit/992c93376c0ed96353e3695f2e5c7d18913d0bb9 because it caused the build system to scream.

**Review by lafriks — APPROVED** [review]

**Comment by jsoref** [comment]
@bzz dropped

**Review by bzz — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Pull request merged**